app/views/application/_footer.erb

```diff
@@ -10,6 +10,9 @@
         <p>
           <a href="/about">About</a>
         </p>
+        <p>
+          <a href="/contribute">Contribute to Exercism</a>
```

Comment:
The current link is right below the `About` link in the footer, the alternative is adding it to `app/views/application/footer/links.erb`. I chose to place it under `About` because placing it under `The Command-Line Client` link felt strange (layout-wise). But since I'm far from being an expert, I really want a second opinion. 😄